.gitignore

```diff
@@ -21,3 +21,4 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+.idea/*
```

Comment:
Please, add a new emptu line at the end of the file

Reply:
Done.